packages/core/src/__test/testTools.ts

```diff
@@ -1,7 +1,46 @@
+import { TolgeePlugin } from '../types';
+
 export const resolvablePromise = <T = any>() => {
   let resolve: (value: T) => void;
   const promise = new Promise<T>((innerResolve) => {
     resolve = innerResolve;
   });
   return [promise, resolve!] as const;
 };
+
+export const DevToolsPlugin =
+  (postfix = ''): TolgeePlugin =>
```

Comment:
```suggestion
  (suffix = ''): TolgeePlugin =>
```

Postfix is not very common form in this context.
https://english.stackexchange.com/questions/81263/postfix-or-suffix